### DISCUSSION TITLE
error-stack: Move `error-stack` to a different repo?

### DISCUSSION THREAD

**Discussion by dpc** [hashintel/hash#2882] [error-stack (Help)] [answered]
It always seemed to me that `error-stack` sitting in a subdirectory of some other project is making it less visible, and hiders ability to build a community around it.

I realize that you've built it to work inside that larger app, but I doubt it would be a problem if it was e.g. https://github.com/hashintel/error-stack .

**Comment by vilkinsons** [comment, marked as answer]
Hi @dpc, thanks for the comment. We've answered this previously (and our position hasn't changed at this time), so I'm going to close this thread. You can find the original question/answer here: https://github.com/hashintel/hash/discussions/686#discussioncomment-3008598
